### ISSUE TITLE
bug: ifelse doesn't seem to work

### ISSUE THREAD

**Issue by wjones127** [ibis-project/ibis-substrait#374]
```python
import ibis
from ibis_substrait.compiler.core import SubstraitCompiler

compiler = SubstraitCompiler()

t = ibis.table([("a", "double"), ("b", "int64"), ("c", "double"), ("d", "string")], name="table0")

query = t.mutate(valid=(t['a'] > 0).ifelse(1, 0))

compiler.compile(query)
```

Produces:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/core.py", line 89, in compile
    input=translate(expr.op(), expr, self),
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/functools.py", line 889, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/translate.py", line 726, in selection
    expressions=[
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/translate.py", line 727, in <listcomp>
    translate(
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/functools.py", line 889, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/translate.py", line 203, in _expr
    return translate(expr.op(), expr, compiler, **kwargs)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/functools.py", line 889, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/translate.py", line 470, in alias_op
    return translate(op.arg.op(), op.arg, compiler, **kwargs)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/functools.py", line 889, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/translate.py", line 483, in value_op
    function_reference=compiler.function_id(expr),
  File "/Users/willjones/mambaforge/envs/substrait-test/lib/python3.10/site-packages/ibis_substrait/compiler/core.py", line 69, in function_id
    op_name = IBIS_SUBSTRAIT_OP_MAPPING[op_type.__name__]
KeyError: 'Where'
```

But in theory this is implemented:

https://github.com/ibis-project/ibis-substrait/blob/9e680ef232055fc243677f36d45b9ac4f8143b52/ibis_substrait/tests/compiler/test_decompiler.py#L274-L275

**Comment by jcrist** [comment]
I believe this was fixed already by #367.

**Comment by gforsyth** [comment]
Closed via #367
